task10/add.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
Having multiple pages goes against SPA pattern thet was in TR. Instead, some DOM manipulations should've been used